processor/k8sprocessor/client_test.go

```diff
@@ -62,6 +65,11 @@ func (f *fakeClient) GetPod(identifier kube.PodIdentifier) (*kube.Pod, bool) {
 	return p, ok
 }
 
+func (f *fakeClient) GetNamespace(namespace string) (*kube.Namespace, bool) {
+	ns, ok := f.Namespaces[namespace]
```

Comment:
could directly return the results

Reply:
@owais Could you elaborate a little more on this one?
I did the same return as the GetPod function above it for consistency.

Reply:
It's not a big deal. I mean you could write it as `return f.Namespaces[namespace]`

Reply:
@owais I think I will skip this one, but I resolved all the other comments. :)

Reply:
OK :)